packages/plugins/rpc-manager/src/lib/eth_signTypedData.ts

```diff
@@ -41,9 +40,8 @@ export default class EthSignTypedData extends RpcModifier {
     this.logger.trace(__(`Original request/response data: ${JSON.stringify({ request: params.request, response: params.response })}`));
 
     try {
-      const accounts = await this.accounts;
       const [fromAddr, typedData] = params.request.params;
-      const account = accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(fromAddr));
+      const account = this.accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(fromAddr));
```

Comment:
Should it be `acc.address || acc` as seen in [other lines](https://github.com/embark-framework/embark/pull/2150/files#diff-7f2d476e5b66f22cdc294475317d84f9R33)?

Reply:
Have removed `acc.address || acc` from the other source as it was not needed.